---
title: Population structure and phenotypic variation of *Sclerotinia sclerotiorum* from dry bean in the United States
preprint: false
author:
  - name: Zhian N. Kamvar
    email: zkamvar@unl.edu
    affiliation: 1
  - name: Bimal Sajeewa Amadarasa
    email: bamadarasa2@unl.edu
    affiliation: 1
  - name: Rachana Jhala
    email: rjhala2@unl.edu
    affiliation: 1
  - name: Serena McCoy
    email: ¯\_(ツ)_/¯
    affiliation: 1
  - name: James Steadman
    email: jsteadman1@unl.edu
    affiliation: 1
  - name: Sydney E. Everhart
    email: everhart@unl.edu
    affiliation: 1
    corresponding: true
affiliation:
  - code: 1
    address: Department of Plant Pathology, University of Nebraska-Lincoln, Lincoln 68583
abstract: >
  `r paste(readLines("abstract.md"), collapse = "\n  ")`
header-includes: >
  \usepackage{indentfirst}
bibliography: ssc_bibliography.bib
output: 
  rticles::peerj_article:
    citation_package: natbib
    base_format: bookdown::pdf_document2 # for using \@ref()
---

```{r setup, include = FALSE, echo = FALSE}
knitr::opts_knit$set(eval.after = c('fig.cap', 'fig.scap'))
knitr::opts_chunk$set(message = FALSE, warning = FALSE)
```

```{r initiate, echo = FALSE, message = FALSE, warning = FALSE}
library("poppr")
load("../../data/sclerotinia_16_loci.rda")
dat11cc <- clonecorrect(dat11, ~Region/Source/Host/Year)

dapc_region      <- dapc(setPop(dat11cc, ~Region), n.da = 13, n.pca = 20)
dapc_region_year <- dapc(setPop(dat11cc, ~Region/Year), n.da = 12, n.pca = 22)
dapc_host <- dat11cc %>% 
  setPop(~Source) %>%
  popsub("wmn") %>%
  setPop(~Host) %>% 
  popsub(c("Beryl", "Bunsi", "G122")) %>%
  dapc(n.da = 2, n.pca = 20)

prettify <- function(x, i) signif((1 - x[i])*100, 3)
paxis <- function(d, i = 1 ){
  res <- cumsum(d$eig)/sum(d$eig)
  prettify(res, i)
}
passp <- function(d, i = 1){
  res <- apply(d$posterior, 2, mean, na.rm = TRUE)
  prettify(1 - res, i)
}
```


# Introduction {-}

<!-- 
ZNK: The introduction needs a lot of work. Here's an outline of what I think
     needs to go in here.
Outline:

 - *S. sclerotiorum* basic biology
 - What have genetic studies and phenotypic studies found
 - What are we missing? A detailed analysis of *S. sclerotiorum* from diverse
   regions that may potentially identify genotypes or MCGs that may be good
   tools for testing breeding efforts.
 - Our hypotheses and specific questions.
 -->

*Sclerotinia sclerotiorum* (Lib.) de Bary is an ascomycete plant pathogen found worldwide.
This is a necrotrophic pathogen that is primarily homothallic (self-fertilization) [@bolton2006sclerotinia; @sexton2006population] that causes disease on more than 400 plant species belonging to 75 families [@boland1994index].
These include crops of major economic importance such as sunflower (*Helianthus spp.*), soybean (*Glycine max* L.), canola (*Brassica napa* L., *Brassica campestris* L.), and dry bean (*Phaseolus vulgaris* L.) [@bolton2006sclerotinia]. 
On dry bean, *S. sclerotiorum* is the causal agent of White Mold, a devastating disease that can be a yield-limiting in temperate climates [@steadman1983white].

<!-- ZNK: TMI -->
<!-- Sclerotia are capable of surviving more than five years in soil [@steadman1983white] and germinate carpogenically or myceliogenically to infect the plant when environmental conditions are favorable.
A carpogenically germinated sclerotia gives rise to apothecia in which ascospores are formed.
Ascospores infect senescent plant tissues that, when supplemented with cool and humid micro-climate within the plant canopy, leads to mycelial infection of the host plant.
Consequently, flowering, which coincide with canopy closure is a critical factor of disease initiation since sensing flower parts fallen on to leaves, petioles or stems provide the required energy source for plant infection.
Myceliogenically germinated sclerotia result in formation of hyphae, which also have potential for directly plant infection, though this is considered rare and less important within the disease cycle of the pathogen [@abawi1975source].
Diseased bean fields initially show scattered wilted leaves.
Symptoms on individual plants are water-soaked lesions on leaves, pods and stems, which expand over time and show signs of white fungal mycelia.
Lesions on stems and branches become necrotic as they age and dead tissues appear bleached.
Sclerotia are often produced on dead or dying plants once conditions become unfavorable for fungal growth. -->

Previous studies of *S. sclerotiorum* population biology in North America and Australia have reported clonal propagation of the pathogen [@cubeta1997clonality; @ekins2011population; @kohli1998random; @kohli1995clonal] with evidence of outcrossing observed in some areas [@atallah2004high; @kohli1998random; @malvarez2007new; @sexton2004microsatellite].
One such study examined *S. sclerotiorum* population structure using microsatellite markers from potato fields in the Columbia Basin of Washington state.
Due to high crop diversity and high presence of the pathogen, it was important to understand the potential for outcrossing in order to develop disease management strategies.
Molecular analysis revealed 92% of the isolate variability was found within subpopulations [@atallah2004high].
However, there was evidence of linkage equilibrium of microsatellite loci in 2 locations and illustrated the possibility of outcrossing in this region.  

<!-- ZNK: This is more discussion-material -->
<!-- A population study from Australian canola fields showed evidence of sexual outcrossing of *S. sclerotiorum* isolates [@sexton2004microsatellite].
Another study characterized three populations of *S. sclerotiorum* from lettuce in California (CA) and Ontario (ON), and pea/lentil in Washington (WA) [@malvarez2007new].
While all three populations were  clonal, the index of association ($I_A$) based on DNA fingerprint was closer to zero for CA than it was for the other populations.
Populations from Australian sunflower with evidence of clonal reproduction were reported, and despite populations sampled from fields separated 50 to 400 km and from both head and basal rots, all showed similar gene diversity, with authors concluding isolates representing different geographies belong to one large genetic population of *S. sclerotiorum* [@ekins2011population].  -->

<!-- SEE: Avoid sentences that start with the authors/reference. -->
@attanayake2013sclerotinia studied population differentiation of *S. sclerotiorum* isolates infecting canola from China and the USA.
There were no shared microsatellite multilocus genotypes (MLG) between the two populations.
Sexual outcrossing was detected in both populations with higher recombination rate in Chinese population than the USA population.
Additionally, phenotypic traits such as production of sclerotia, mycelial compatibility group (MCG), growth rate were also significantly different between the two populations.
Genetic diversity of *S. sclerotiorum* from soy bean, canola, sunflower, and dry bean in the North Central region of the USA was studied by [@aldrich-wolfe2015genetic].
Microsatellite analysis revealed little host- specificity or genetic-isolation of *S. sclerotiorum* isolates among host crops as 93% of the observed genetic variation was explained by within-host differences.
Isolates from all host crops were primarily reproducing clonally while there were also evidence of genetic recombination of isolates from canola and sunflower.

<!-- Most of the previous studies on *S. sclerotiorum* had been conducted in the USA, Canada and Australia but not in Europe. 
In those studies, different dry bean cultivars had been rarely used as  hosts to investigate any host-pathogen relationships or specificity.
No white mold resistant dry bean cultivars exist.
Canopy architecture and growth pattern of some cultivars enable improved disease avoidance [@kolkman2002agronomic]. -->
To date, no commercial white mold-resistant dry bean cultivars exist. 
Field trials in some regions of the USA have shown the G122 cultivar is partially resistant while the Beryl cultivar is susceptible [@otto-hanson2011variation; @steadman2004identification].
<!-- This puny paragraph needs a lot of work to more fully describe the operation of the wmn -->

Understanding genetic relationships and reproduction behavior of *S. sclerotiorum* populations from locations worldwide is beneficial for plant breeders seeking to develop white mold resistance cultivars for deployment worldwide. 
This information is also helpful in making disease management decisions and developing disease resistance screening methods.

<!-- Microsatellite markers (simple sequence repeats or SSR) are codominant, polymorphic and also able to detect multiple alleles per locus [@atallah2004high; @sirjusingh2001characterisation].
Reproducibility of microsatellites is higher compared other fingerprinting techniques such as random amplified polymorphic DNA (RAPD) [@atallah2004high].
High mutagenic rate and multiallelic nature of microsatellites render suitability for inferring population structure of organisms and their potential for outcrossing.
In the case of plant pathogens like *S. sclerotiorum* that is predominantly clonal and occasionally outcrossing, microsatellites have been used successfully to ascertain their ability to reproduce by sexual outcrossing [@atallah2004high; @sexton2004microsatellite] and also to detect differences among clonally propagated individuals.  -->

In this study, we analyze and characterize the genetic and phenotypic diversity of 366 *S. sclerotiorum* isolates collected between 2003 and 2012 from dry bean cultivars among different geographic locations in the United States, France, Mexico, and Australia. 
We wanted to know if the *S. sclerotiorum* populations from white mold screening nurseries were representative of the fields within the same region. 
As these nurseries are not treated with any chemical or cultural control of white mold, we hypothesized that these nurseries would represent the natural population of *S. sclerotiorum*.
Furthermore, we wanted to investigate the potential effect of cultivar on genetic diversity by assessing three dry bean cultivars with different levels of resistance, Beryl (great northern bean, susceptible), Bunsi (navy bean, low susceptibility), and G122 (cranberry bean, partial resistance) [@otto-hanson2011variation].
We additionally wanted to determine categorical or phenotypic variables that best predicted genetic structure and if there was correlation between multilocus genotype and mycelial compatibility group.
By investigating these aims, we hope to effectively describe the population structure of *S. sclerotiorum* in the USA and make available our database of isolates for use in future dry bean breeding efforts. 


# Materials and Methods {-}

## Isolate collection {-}

Several of the isolates used for this study were collected and reported in previous studies and were collected using the same methods [@otto-hanson2011variation].
Broadly, isolates were collected from two sources, white mold screening nurseries (wmn) or producer fields.
The white mold screening nurseries were 5m x 10m in size and maintained without applying fungicides to observe natural incidence of white mold.
The nursery plots were incorporated with a basal dressing of N:P:K = 1:3:2 and side dressing of 0:3:2 during the growing season [@steadman2003identification; @steadman2004identification; @steadman2005identification; @steadman2006identification; @otto2007identification].
Sampling was carried out by collecting diseased tissue in zig-zag transects across field plots. 
Because sampling depended on disease incidence, the number of samples isolated varied from year to year.
Although the nursery locations were the same over sampling years, sampling plots within a location varied for sampling years.

Sclerotia of *S. sclerotiorum* were collected over several years from grower fields and/or wmn in 11 states of the United States of America, Mexico, France, and Australia.
The white mold screening nurseries in different regions are indicated by wmn and all other isolates came from commercial fields (Table \@ref(tab:isolate-table)). 

After collection, the isolates were stored in petri plates lined with filter paper, then stored at $20^{\circ}$F or -$4^{\circ}$C. 
Sclerotia were triple rinsed with (i) 50% Clorox bleach/50% double-distilled (dd)H2O solution for 3 min, (ii) ddH2O rinse for 3 min, and (iii) final ddH2O rinse for 3 min, then plated on water agar plates (16g of Bacto agar per liter of ddH2O), with four to five sclerotia of each isolate separated on each plate and stored on the bench top at room temperature for 5 to 6 days An 8-mm plug from a 5- or 6-day-old culture was transferred from the advancing margin of the mycelia onto a plate of Difco potato dextrose agar (PDA at 39 g/liter of ddH2O) [@otto-hanson2011variation].
In combination with the 156 isolates described previously, we collected 210 isolates for a total of 366 isolates [@otto-hanson2011variation].

## Mycelial Compatibility {-}

MCG was determined as described previously through co-culturing pairs of 2-day-old isolates 2.5cm apart on Diana Sermons (DS) Medium [@cubeta2001mycelial]. 
Incompatibility of different MCGs resulted in formation of a mycelial barrier of dead tissue (termed a barrage line) accompanied by formation of sclerotia on either side of the barrage line, indicating the limits of the isolates' growth [@otto-hanson2011variation; @kohn1990mycelial; @leslie1993fungal].
Isolates were compared in a pairwise manner for each site and then representatives among sites were compared to determine mycelial compatibility groups by scoring compatible and incompatible interactions [@otto-hanson2011variation]. 

<!-- ![
Examples of mycelial compatibility results in *S. sclerotiorum*. 
A) Compatible reaction showing aerial hyphae.
B) Incompatible reaction showing barrage reaction of dead cells at point of contact.
](../../results/figures/publication/incompatibility2.JPG){#incompatibility width=25%} -->

## Aggressiveness {-}

Aggressiveness of each isolate was assessed using a straw test as described in @otto-hanson2011variation that rated necrotic lesion size [@petzoldt1996straw; @teran2006modified].
Briefly, the straw test uses 21-day-old G122 plants as the host in a greenhouse setting. 
Clear drinking straws cut to 2.5cm and sealed were used to place two mycelial plugs of inoculum on the host plant after removing all plant growth &gt;2.5cm above the fourth node.
Measurements were taken after 8 days using the Modified Petzoldt and Dickson scale where a score of 1--9 where 1 is no disease and 9 is plant death [@petzoldt1996straw; @teran2006modified].

## Microsatellite genotyping {-}

Prior to DNA extraction, isolates were grown on PDA and plugs were subsequently transferred to Potato Dextrose Broth (PDB) where they were grown until there was significant mycelial growth, but before the mycelial mat became solidified (4--5 days).
Mycelial was collected in a buchner funnel, agar plugs removed, lyophilized and pulverized manually in whirl-pak bags. 
Lyophilized mycelia was then stored in microcentrifuge tubes at -$20^{\circ}$C until needed for DNA extraction.
DNA from 25mg of pulverized mycelia was purified using a phenol-chloroform extraction method followed by alcohol precipitation, suspending the DNA in 200$\mu$l TE [@sambrook1989molecular].
Suspended DNA was stored at $4^{\circ}$C until genotyping.

We genotyped *S. sclerotiorum* isolates using 16 microsatellite primer pairs developed previously [@sirjusingh2001characterisation].
PCR was carried out as described using primers labeled with FAM fluorophore, and resulting amplicons were first resolved in a 1.5% agarose gel stained with ethidium bromide to ensure product was within the expected size range prior to capillary electrophoresis.

Capillary electrophoresis (fragment analysis) of amplicons, with size standard GeneScan™ 500 LIZ®, was performed using an ABI 3730 genetic analyzer (Life Technologies Corporation, Carlsbad, CA) at the Michigan State University Genomic Sequencing Center. 
Alleles were scored using PeakScanner version 1.0 (Life Technologies Corporation, Carlsbad, CA) and recorded manually in an excel sheet.

## Data processing and Analysis{-}

All analyses were performed on this data set unless otherwise mentioned.
All data processing and analyses were performed in R version 3.4.1 [@R] and are openly available and reproducible at https://github.com/everhartlab/sclerotinia-366/.
Of the 16 microsatellite loci genotyped, five included compound repeats, which made it challenging to accurately/confidently bin alleles into fragment sizes expected for each locus based on the described repeat motif. 
<!-- add something about repeat length consistency  -->
Loci with compound repeats were removed for the reported statistics, but were processed and included in data analyses to ensure the integrity of the results with 11 loci as compared to 16.
We assessed the power of our 11 markers by generating a genotype accumulation curve in the R package *poppr* version 2.4.1, looking for evidence of saturation, which would indicate that loci were sufficiently sampled to adequately represent the full set of genotypes [@arnaud2007standardizing; @kamvar2015novel].
To avoid including isolates potentially collected from the same plant (which increases the probability of collecting sclerotioa from the same point of infection more than once), data were clone-corrected on a hierarchy of region/source/host/year---meaning that one representative of each genotype was retained for each combined regions, source, host, and year---for subsequent analysis. 
We assessed genotypic diversity by calculating Stoddart and Taylor's index (*G*) [@stoddart1988genotypic], Shannon's index (*H*) [@shannon2001mathematical], Simpson's index ($\lambda$) [@simpson1949measurement], and evenness ($E_5$, *eMLG*) [@heck1975explicit; @hurlbert1971nonconcept; @pielou1975ecological; @grunwald2003analysis]. 
To assess the potential for random mating, we tested for linkage disequilibrium with the index of association, $I_A$ and its standardized version, $\bar{r}_d$ using 999 permutations [@agapow2001indices; @smith1993how; @brown1980multilocus].
Both genotypic diversity and linkage disequilibrium were calculated in *poppr* version 2.4.1 [@kamvar2014poppr].

## Assessing Importance of Variables  {-}

### Distance-based Redundancy Analysis {-}

A distance-based redundancy analysis (dbRDA) [@legendre1999distance] was performed with the function `capscale()` in the *vegan* package version 2.4-3 [@vegan].
This method uses constrained ordinations on a distance matrix representing the response variable to delineate relative contribution of any number of independent explanatory variables. 
We used this method to delineate the phenotypic (Aggressiveness, Mycelial Compatibility Group (MCG)), geographic (Region, Host, Location), and temporal (Year) components in predicting genetic composition of the populations.
<!-- RDA is a multivariate linear regression analysis of independent (explanatory: eg. environmental and phenotypic) variables on dependent (non-explanatory or response) variables. -->
The distance matrix we used as the response variable was generated using Bruvo's genetic distance from clone-corrected data (procedure described above) as implemented in *poppr* version 2.4.1, which employed a stepwise mutation model for microsatellite data [@bruvo2004simple; @kamvar2014poppr]. 
Because aggressiveness measures differed with each isolate, aggressiveness first averaged across clone-corrected isolates.
<!-- In our case, a pairwise Bruvo's [@bruvo2004simple] distance matrix was generated from microsatellite data using *poppr* 2.4.1 [@kamvar2014poppr; @kamvar2015novel] and employed it as the dependent matrix for RDA analysis. -->
To identify explanatory variable(s) correlated with genetic variation, a forward-backward selection process was applied with the vegan function `ordistep()` to identify the set of variables that explained more variation than predicted with the global model containing all variables.
An ANOVA was then performed to test for significance of the reduced model and marginal effects using 999 permutations.
The `varpart()` function of vegan was used to determine variation partitioning of explanatory variables.

### Aggressiveness Assessment {-}

We used analysis of variance (ANOVA) to assess if aggressiveness could be explained by Region, MCG, or multilocus genotype (MLG). 
To minimize complications due to small sample sizes, we chose the top 10 MCGs, representing 56.5% of the isolates collected, the top 10 MLGs representing 26.7% of the isolates, and populations with more than five isolates.
Between-group differences were assessed using Tukey's HSD test with an alpha of 0.05 using the `HSD.test()` function in the package *agricolae* version 1.2.4 [@mendiburu2015agricolae].

### Correlating Multilocus Genotypes with Mycelial Compatibility Groups {-}

We wanted to assess if there was correlation between MLGs and MCGs. 
In simple terms, this was done using a network-based approach where both MLGs and MCGs were considered nodes and the number of isolates in which they were found together was the strength of the connection between an MLG and and MCG node.
The network-based approach allowed us to assess how "clumpy" the relationships between MLGs and MCGs were.
In more specific terms, we first created a contingency table with MLGs and MCGs and converted it to a directed and weighted edgelist where each edge was a connection from an MCG to an MLG weighted by the number of samples shared in the connection. 
This was then converted to a bipartite graph where top nodes represented MCGs and bottom nodes represented MLGs.
The cluster walktrap community detection algorithm as implemented in the `cluster_walktrap()` function in *igraph* version 1.1.2 was used to define clusters of MLGs and MCGs [@csardi2006igraph; @pons2006computing].
All communities that had fewer than 10 members were then consolidated into one.
Community definitions were used to assess the average genetic distance (as defined by Bruvo's distance) within members of the community [@bruvo2004simple].

## Genetic Diversity {-}

### Population Differentiation {-}

We used analysis of molecular variance (AMOVA) with Bruvo's genetic distance in *poppr* version 2.4.1 to test for differentiation associated with wmn by quantifying the differentiation conferred by Region, Source, and Year [@excoffier1992analysis; @bruvo2004simple; @kamvar2014poppr].
Source in this case was defined as a population belonging to a wmn or not. 

To identify specific Regions that were more differentiated than others, we used discriminant analysis of principal components (DAPC) as implemented in *adegenet* version 2.1.0, assessing the per-sample posterior group assignment probability [@jombart2008adegenet]. 
This method decomposes the genetic data into principal components, and then uses a subset of these as the inputs for discriminant analysis, which attempts to minimize within-group variation and maximize among-group variation [@jombart2010discriminant]. 
To avoid over-fitting data, the optimal number of principal components was selected by using the *adegenet* function `xvalDapc()`. 
This function implements a cross-validation procedure to iterate over an increasing number of principal components on a subset (90%) of the data, trying to find the minimum number of principal components that maximizes the rate of successful group reassignment. 
To assess if cultivar had an influence on genetic diversity between wmn, we first subset the clone-corrected data to contain only samples from wmn and from the cultivars Beryl, Bunsi, and G122 and tested differentiation using AMOVA and DAPC as described above. 
We additionally assessed population stability over time by calculating DAPC over the combined groups of Region and Year as described above.

### Analysis of Shared Multilocus Genotypes {-}

We wanted to evaluate patterns of connectivity between shared multilocus genotypes across geographic regions.
We first tabulated the multilocus genotypes shared between at least two populations (defined as states or countries) with the *poppr* function `mlg.crosspop()` [@kamvar2014poppr].
From these data, we constructed a graph with populations as nodes and shared genotypes as edges (connections) between nodes using the R packages *igraph* version 1.0.1 [@csardi2006igraph], *dplyr* version 0.7.1 [@dplyr], and *purrr* version 0.2.2.2 [@purrr].
Each node was weighted by the fraction of shared MLGs.
Each edge represented a single MLG, but because a single MLG could be present in more than one population, that MLG would be represented by (*n*\*(*n*-1))/2 edges where n represents the number of populations crossed.
Edges were weighted by $1-P_{sex}$, where $P_{sex}$ is the probability of encountering the same genotype via two independent meiotic events [@arnaud2007standardizing; @parks1993study].
We then used the `cluster_optimal()` function from *igraph* to define communities [@csardi2006igraph].
The graph was plotted using the R packages *ggplot2* version 2.2.1 [@ggplot2] and *ggraph* version 1.0.0 [@ggraph].
To ensure that we captured the same community signal, we additionally performed this analysis including the five polymorphic markers described above.

# Results {-}

A total of 366 isolates were collected from 2003 to 2012 (except 2006 and 2011) from diseased dry bean plants in Australia, Mexico, France, and 11 states in the USA (Table \@ref(tab:isolate-table)).
Over 50% of the isolates came from four states, MI (62), ND (60), WA (59), NE (47). 
Four regions had fewer than 10 isolates, Australia (6), WI (2), NY (1), ID (1).
We observed 87 MCGs, the most abundant of which ('MCG 5') was represented by 73 isolates over 37 MLGs (Fig \@ref(biggraph)A,C).
After clone-correction on the hierarchy of Region/Source/Host/Year, we were left with 318 isolates representing 165 MLGs (Table \@ref(tab:iatab)).
Analysis of the genotype accumulation curve showed no clear plateau (Supplemental Information), indicating that we would likely obtain more multilocus genotypes if we were to genotype more loci. The number of observed alleles per locus ranged from two to 10 with an average of 6.27 (Table \@ref(tab:locus-stats)).

-----------------------------------------------------------------------------
Country   State  Field Code              Year Host                          N 
--------- ------ ---------- ----------------- -------------------------- ----
USA       CA     wmn               2004, 2005 Beryl, Bunsi, G122           18 

USA       CO     producer          2007, 2010 Pinto, Yellow                41 

                 wmn                     2003 GH                            1 

USA       ID     producer                2003 GH                            1 

USA       MI     wmn        2003, 2004, 2005, 11A, 37, 38, B07104,         43 
                                   2008, 2009 Beryl, Bunsi, Cornell,       
                                              G122, Orion, PO7863, WM31      

                 producer    2003, 2008, 2009 BL, Black, Fuji, GH,         19 
                                              Merlot, SR06233, unk,        
                                              Vista, Zorro 

USA       MN     wmn               2003, 2004 Beryl, Bunsi, G122           11 

USA       ND     producer          2007, 2010 unk                          53 

                 wmn                     2005 Beryl, Bunsi, G122            7 

USA       NE     wmn        2004, 2005, 2008, Beryl, Bunsi, G122,          27 
                                         2010 PO7683, unk                    

                 producer   2003, 2007, 2009, Beryl, Emerson, GH, Orion,   20 
                                         2010  Pinto, Weihing                 

USA       NY     producer                2003 GH                            1 

USA       OR     wmn               2003, 2004 Beryl, Bunsi, G122           15 

                 producer                2003 G122, GH                      2 

USA       WA     wmn        2003, 2004, 2005, 11A, 37, 38, Beryl, Bunsi,   36 
                                         2008  Cornell, G122, Orion, PO7      
                                              104, PO7863, WM31               

                 producer          2003, 2007 GH, Merlot, Pinto, Redkid    23 

USA       WI     producer                2003 GH                            2 

Mexico    -      wmn                     2005 Beryl, Bunsi, G122           18 

France    -      wmn               2004, 2005 Beryl, Bunsi, G122           18 

                 producer                2012 unk                           4 

Australia -      wmn                     2004 Beryl, Bunsi, G122            4 

                 producer                2004 Beryl                         2 

-----------------------------------------------------------------------------

Table: (\#tab:isolate-table) Description of *Sclerotinia sclerotiorum* isolates used in this study.
N = Number of Isolates. 
Key abbreviations: wmn = white mold screening nursery, producer = producer field, unk = unknown cultivar.


```{r , echo = FALSE, warning = FALSE, message = FALSE, results = "asis"}
suppressPackageStartupMessages(library("tidyverse"))
loc_columns <- cols(
  Locus = col_character(),
  Range = col_character(),
  Repeats = col_double(),
  allele = col_integer(),
  Hexp = col_double(),
  Evenness = col_double()
)
loc_table <- readr::read_csv("../../results/tables/locus-table.csv", 
                             col_types = loc_columns,
                             col_names = TRUE)
loc_table %>%
  mutate(allele = round(allele, 2)) %>%
  mutate(Repeats = case_when(
    Repeats <= 2 ~ "di-",
    Repeats <= 3 ~ "tri-",
    Repeats <= 4 ~ "tetra-",
    Repeats <= 5 ~ "penta-",
    Repeats <= 6 ~ "hexa-"
  )) %>%
  rename(`*h*` = Hexp) %>%
  rename(`No. alleles` = allele) %>%
  rename(`Repeat Motif` = Repeats) %>%
  huxtable::as_huxtable(add_colnames = TRUE) %>%
  huxtable::set_number_format(huxtable::everywhere, 3:4, 0) %>%
  huxtable::set_number_format(huxtable::everywhere, 5:6, 2) %>%
  huxtable::set_align(huxtable::everywhere, 2:6, "right") %>%
  huxtable::set_col_width(c(0.06, 0.07, 0.11, 0.1, 0.05, 0.075)) %>%
  huxtable::print_md(max_width = 61)

Amean <- mean(loc_table$allele)
Hmean <- mean(loc_table$Hexp)
Emean <- mean(loc_table$Evenness)
```

Table: (\#tab:locus-stats) Allelic diversity on full data set at loci used in this study. *h* = Nei's Gene Diversity [@nei1978estimation]. Average *h* = `r signif(Hmean, 3)`, average Evenness = `r signif(Emean, 3)`, average no. alleles = `r signif(Amean, 3)`

## Variable Assessment {-}

### Variable Importance {-}

The forward-backward selection process of the dbRDA models on clone-corrected data revealed Year, Region, Host, and Aggressiveness to be the optimal variables for the reduced model, accounting for 45% of the total variation.
ANOVA revealed that the reduced model was significant with an adjusted R^2^ of 0.0674 (*P* = 0.001).
Assessment of the marginal effects showed that region, host, year, and aggressiveness significantly explained genetic variation (*P* = 0.001, *P* = 0.009 for region).
It was revealed that there was multicollinearity when MCG was combined with any other variable, so we followed the same procedure, dropping MCG from the list of potential predictors. 
From these results, we found that aggressiveness, year, region, and host were optimal, accounting for 17.6% of the total variation.
ANOVA revealed significant effects with an adjusted R^2^ of 0.0325 (*P* = 0.001).
While the marginal effect assessment showed that region, host, and year all significantly explained the variation at *P* = 0.001, and aggressiveness significantly explained the variation at *P* = 0.045.
Much of the variation appears to be driven by isolates from Mexico and 2005 (Fig. \@ref(rda-biplot)).
Variance partitioning of the independent variables without MCG indicated aggressiveness to be the least influential factor with 0.1% contributing to explaining the variation of molecular data whereas the combination of variables accounted for 3.3%.

<!-- RDA figure -->

![
Biplot showing five most influential explanatory variables (arrows) overlayed on the first two eigenvectors of distance based redundancy analysis of *S. sclerotiorum* isolates. 
The length of the arrows are directly proportional to the strength of the correlation between explanatory and molecular variables. 
Open circles represent the 318 clone-corrected samples in ordination space. 
](../../results/figures/publication/Figure7Z.pdf){#rda-biplot width=61%}

<!-- end figure -->

### Aggressiveness {-}

<!-- aggressiveness figure -->

![
Boxplot representing aggressiveness (vertical axis) by population (horizontal axis) arranged in descending order of mean aggressiveness for all populations with N > 5. Box boundaries represent 25% and 75% percentiles and whiskers extend to at most 1.5 $\times$ the interquartile range. Horizontal line represents median value. Circles represent individual observations where filled circles indicate isolates from white mold screening nurseries (wmn).
](../../results/figures/publication/aggressiveness.pdf){#aggressiveness-boxplot}

<!-- end figure -->

Our assessment of aggressiveness in association with Region showed a significant effect (P < 1.00e^-4^), with means that ranged from 5.8 (MN) to 4.0 (CA) (Fig. \@ref(aggressiveness-boxplot), Table \@ref(tab:region-aggressiveness)). 
MCGs also showed a significant effect (P < 1.00e^-4^), with means that ranged from 6.0 (MCG 44) to 4.6 (MCG 49).
We additionally found a significant effect for MLGs (*P* = 7.44e^-4^) for MLGs, with means that ranged from 6.0 (MLG 78) to 4.3 (MLG 140).

### Correlation of Mulitlocus Genotypes and Mycelial Compatibility Groups {-}

Our network-based approach to correlating MLGs with MCGs revealed a large and complex network (Fig. \@ref(biggraph)). 
Community analysis showed 51 communities, 15 of which consisted of a single MLG unconnected with any other community indicating that just 9.09% of the 165 MLGs are unable to cross with any other MLG in this data set (Fig. \@ref(fullgraph)).
The three communities with the most members contained eight of the 10 most abundant MCGs. 
Comparing these communities with Bruvo's genetic distance revealed an average distance of 0.451 among communities and an average distance of 0.437 within communities, showing no significant difference.
When we assessed the number of times two different MLGs that are in the same MCG, considering these as potential heterothallic pairings that could result in sexual recombination, we found an average of 16.5 potential heterothallic parings per MLG. 
Representing just four isolates, 'MLG 75' had 57 neighbors that shared the same mycelial compatibility group (Fig \@ref(biggraph), \@ref(fullgraph)). 
Overall, there was no clear pattern to the association between MLG and MCGs. 


<!-- MCG graph figure -->

![
Data show complex associations between Mycelial Compatibility Groups and Multilocus Genotypes.
**A)** Barplot of Mycelial Compatibility Group (MCG) abundance in descending order.
Singletons (46) were truncated, leaving 41 MCGs.
White bars represent sample counts and grey bars represent counts of unique multilocus genotypes (MLG).
The transparency of the bars represent the evenness of the distribution of the MLGs within a given MCG.
A dashed box surrounds the 8 most common MCGs representing >51% of the data.
**B)** Full graph-representation of the relationship between MCGs (open circles) and MLGs (filled circles).
Details in fig. \@ref(fullgraph). 
**C)** A subset of **B** representing the 8 most common MCGs and their associated MLGs (dashed box in **A**).
Filled nodes represent MLGs and open nodes represent MCGs.
Numbers inside nodes are the MLG/MCG label. 
Node (circle) area scaled to the number of samples represented (range: 1--73).
Edges (arrows) point from MLG to MCG where the weight (thickness) of the edge represents the number of shared isolates (range: 1--19).
Edges extending from MLGs displayed to other MCGs are not shown.
](../../results/figures/publication/mcg-subgraph-with-context.pdf){#biggraph width=100%}

<!-- end fig -->


## Structure of Shared Multilocus Genotypes {-}

```{r table_ia, echo = FALSE, warning = FALSE, message = FALSE, results = "asis"}
suppressPackageStartupMessages(library("tidyverse"))
read_csv("../../results/tables/genotype_diversity_table_region.csv", 
         na = "NA", # spaces read in as spaces
         col_types = cols(
         Pop = col_character(),
         N = col_integer(),
         MLG = col_integer(),
         eMLG = col_double(),
         SE = col_double(),
         H = col_double(),
         G = col_double(),
         lambda = col_double(),
         E.5 = col_double(),
         Hexp = col_double(),
         Ia = col_double(),
         p.Ia = col_double(),
         rbarD = col_double(),
         p.rD = col_double(),
         File = col_character()
      )) %>%
  # mutate(Ia = case_when(
  #   !is.finite(p.Ia) ~ "-",
  #   p.Ia == 0.001 ~ paste0(round(Ia, 2), "*"),
  #   p.Ia <= 0.01  ~ paste0(round(Ia, 2), "-"),
  #   p.Ia <= 0.05  ~ paste0(round(Ia, 2), "~"),
  #   TRUE          ~ as.character(round(Ia, 2))
  # )) %>%
  mutate(rbarD = case_when(
    !is.finite(p.rD) ~ "-",
    p.rD == 0.001 ~ paste0(round(rbarD, 2), "*"),
  #   p.rD <= 0.01  ~ paste0(round(rbarD, 2), "-"),
  #   p.rD <= 0.05  ~ paste0(round(rbarD, 2), "~"),
    TRUE          ~ as.character(round(rbarD, 2))
  )) %>%
  mutate_if(is.numeric, round, digits = 2) %>%
  mutate(N = paste(N, MLG, sep = " (") %>% paste0(")") %>% formatC()) %>%
  mutate(eMLG = paste(eMLG, SE, sep = " (") %>%  paste0(")") %>% formatC()) %>%
  rename(`*h*` = Hexp) %>%
  rename(`$E_5$` = E.5) %>%
  # rename(`$I_A$` = Ia) %>%
  rename(`$\\lambda$` = lambda) %>%
  rename(`$\\bar{r}_d$` = rbarD) %>%
  select(-File, -MLG, -SE, -p.Ia, -p.rD, -Ia) %>% 
  huxtable::as_huxtable(add_colnames = TRUE) %>%
  huxtable::set_number_format(huxtable::everywhere, 4:5, 1) %>%
  huxtable::set_align(huxtable::everywhere, -1, "right") %>%
  huxtable::set_col_width(c(1.1, 0.9, 1.3, 0.5, 0.5, 1.1, 0.7, 0.6, 0.7)) %>%
  huxtable::print_md(max_width = 80)
```

Table: (\#tab:iatab) Genotypic diversity and Linkage Disequilibrium summary for geographic populations arranged by abundance after clone-correction by a hierarchy of Region/Source/Host/Year. 
Pop = Population, N = number of individuals (number of MLG in parentheses), 
eMLG = expected number of MLGs based on rarefaction at min(10, N) individuals (standard error in parentheses), 
H = Shannon-Weiner Index, G = Stoddardt and Taylor's Index, 
$\lambda$ = Simpson's Index, 
*h* = Nei's 1978 gene diversity, 
$E_5$ = Evenness,
$\bar{r}_d$ = standardized index of association.
An asterix indicates a significant value of $\bar{r}_d$ after 999 permutations, $P \leq 0.001$.

We observed a total of 165 MLGs (215 with 16 loci).
The most abundant MLG was represented by 27 isolates (Table \@ref(tab:mlg-table)) from five regions (NE, MI, WA, CO, and ND). 
Within regions, the genotypes were relatively evenly distributed with moderate to high diversity (Table \@ref(tab:iatab)). 
Of the 165 MLGs, 76 (46%) were found in at least two geographic regions except those found in WI (2), ID (1), and Mexico (18) (Fig. \@ref(community-graph)).
Community analysis of the MLGs shared between populations revealed 4 communities with a modularity of 0.17 (Fig. \@ref(community-graph)): A coastal community (CA, OR, WA, and NY), a midwest community (CO, NE, ND, MI), and an international community (MN, France, Australia).
Analysis with 16 loci resulted in the removal of the NY node because it no longer shared a genotype with OR, but the same overall community structure was present with a modularity of 0.2 (Fig. \@ref(community-graph16)).
The international community appears to be driven by MLG 4, which is shared between all three populations and has a $P_{sex}$ value of 2.87e^-5^, in contrast to the abundant MLG 25, which has a $P_{sex}$ value of 0.0168.

```{r table3, echo = FALSE, warning = FALSE, message = FALSE, results = "asis"}
suppressPackageStartupMessages(library("tidyverse"))
read_csv("../../results/tables/mlg_table.csv", 
         na = "NA", # spaces read in as spaces
         col_types = cols(.default = col_character())) %>%
  rename(`$P_{sex}$` = Psex) %>% 
  knitr::kable(format = "pandoc")
```

Table: (\#tab:mlg-table) The 5 most abundant Multilocus Genotypes (MLG) with
the probability of second encounter ($P_{sex}$), Mycelial Compatibility Groups (MCG), and Regions with sample sizes in parentheses.

<!-- community graph figure -->

![
Network of populations (nodes/circles) and their shared multilocus genotypes (MLG) (edges/lines) genotyped over 11 loci. 
Each node is labeled with **name (number of MLGs shared/number of MLGs total).** 
The shade and area of the nodes are proportional to the number of unique MLGs within the node and the inner nodes are proportional to the number of private MLGs to the region (bottom legend). 
Each edge represents a single MLG where its thickness represents the number of populations that share the MLG and the shade represents the value of $P_{sex}$, or the probability of encountering that MLG from two independent meiotic events.
](../../results/figures/publication/mlg-11.pdf){#community-graph width=100%} 

<!-- end fig -->

\newpage 

## Population Differentiation {-}

### Analysis of Molecular Variance {-}

The analysis of molecular variance (AMOVA) over the hierarchy of Region, Source, and Year showed significant variation between regions and years, but no significant variation between wmn and non-wmn (Table \@ref(tab:amova)).
In contrast, when we compared the three cultivars, Beryl, Bunsi, and G122, we found no significant differentiation (Supplementary Data).  


```{r table4, echo = FALSE, warning = FALSE, message = FALSE, results = "asis"}
suppressPackageStartupMessages(library("tidyverse"))
read_csv("../../results/tables/AMOVA-year.csv", 
         na = "NA", # spaces read in as spaces
         col_types = cols(.default = col_character()))  %>%
  rename(ps = `Phi statistic`) %>%
  # mutate_if(is.numeric, format, digits = 3) %>%
  mutate(ps = signif(as.numeric(ps), digits = 3)) %>%
  mutate(ps = gsub("0\\.00(\\d{1})(\\d{2})", "\\1.\\2e^-3^", ps)) %>%
  mutate(ps = case_when(P > 0.05 ~ ps, TRUE ~ paste0("**", ps, "**"))) %>%
  rename(`$\\Phi$ statistic` = ps) %>%
  rename(`% variation` = `Percent variation`) %>%
  rename(S.S. = `Sum of Squares`) %>%
  rename(`*P*` = P) %>%
  select(c(1:4, 6, 5)) %>%
  huxtable::as_huxtable(add_colnames = TRUE) %>% 
  huxtable::set_col_width(c(0.8, 0.15, 0.15, 0.35, 0.5, 0.15)) %>% 
  huxtable::set_align(huxtable::everywhere, 2:6, "right") %>% 
  huxtable::set_number_format(huxtable::everywhere, 2, 0) %>%
  huxtable::set_number_format(huxtable::everywhere, 6, 3) %>%
  huxtable::set_number_format(huxtable::everywhere, 5, 4) %>%
  huxtable::print_md(max_width = 80)
```

Table: (\#tab:amova) Analysis of Molecular Variance on Bruvo's genetic distance showing little differentiation between white mold screening nurseries (wmn) and other sources. The hierarchy was constructed as Source/Region where source is defined as belonging to a wmn or not. Bold $\Phi$ values indicate significant values (*P* < 0.05). S.S. = Sum of Squares, d.f. = degrees of freedom.

### Discriminant Analysis of Principal Components {-}

Discriminant Analysis of Principal Components (DAPC) was performed grouping by Region with the first `r dapc_region$n.pca` principal components, representing `r signif(dapc_region$var*100, 3)`% of the total variance. 
The first discriminant axis (representing `r paxis(dapc_region)`% of the discriminatory power) separated the centroid for the Mexico isolates from the rest of the data, indicating strong differentiation (Fig. \@ref(DAPC)b). 
The second discriminant axis, representing `r paxis(dapc_region, 2)`% of the discriminatory power, separated the centroid for the California isolates.
The mean population assignment probabilities for all populations with n > 10 showed that only isolates from Mexico, CA, and France had > 50% probabilities of being reassigned to their source populations (Fig. \@ref(DAPC)a). 
DAPC grouping by cultivar used the first 20 principal components, representing 89% of the total variance.
The first two discriminant axes failed to separate any of the cultivars where the mean posterior assignment probabilities were `r passp(dapc_host, "G122")`% (G122), `r passp(dapc_host, "Beryl")`% (Beryl), and `r passp(dapc_host, "Bunsi")`% (Bunsi).
DAPC grouping by Region and Year used the first `r dapc_region_year$n.pca` principal components, representing `r signif(dapc_region_year$var*100, 3)`% of the total variance.
The Midwestern populations (NE, MI, CO, ND) did not appear to have any variation across time in contrast to WA, which showed a shift in population structure in the last year of sampling, 2008 (Fig. \@ref(DAPC-RY)).
Further analysis of this population revealed that all 12 isolates in WA circa 2008 originated in a wmn and nine of these genotypes were shared with CA, and three were shared with France (Fig. \@ref(community-graph), \@ref(community-graph16)).


<!-- DAPC figure -->

![
Discriminant Analysis of Principal Components on regions showing that Mexico is differentiated from other populations. 
**A)** Mean population assignment (horizontal axis) probability (vertical axis) for all populations with N > 10 (facets).
Numbers next to source populations indicate population size.
All values sum to one.
**B)** Scatter plot of first two components from DAPC.
Points represent observed individuals connected to the population centroids with ellipses representing a 66% confidence interval for a normal distribution. 
The center of each component is represented as black grid lines.
](../../results/figures/publication/DAPC.pdf){#DAPC width=100%}

<!-- end figure -->

<!-- DAPC figure 2: Electric Boogaloo-->

![
Scatter plot of Discriminant Analysis of Principal Components on Regions and Years showing temporal variation in WA and CA populations as compared to Midwestern Regions, NE and MI. 
Points (text labels) represent observed individuals connected to the population centroids with ellipses representing a 66% confidence interval for a normal distribution. 
The center of each component is represented as black grid lines.
A more detailed view can be seen in Fig. \@ref(DAPC-RY-FULL).
](../../results/figures/publication/dapc_region_year_micanewa.pdf){#DAPC-RY width=61%}

<!-- end figure -->

### Linkage Analysis {-}

Nearly all populations showed evidence for linkage (Table \@ref(tab:iatab)), which serves as evidence for clonal reproduction or other forms of non-random mating. 
The only exceptions are CA and Australia. Both of these populations showed only moderate significance (P < 0.05) with $\bar{r}_d$ values of 0.03 and 0.12, respectively. 

# Discussion {-}

<!-- 
 - Big Picture reminder of importance
   + This is an extension of a previous work [@otto-hanson2011variation]
   + white mold screening nurseries serve as good way to screen potential resistant lines, avoiding environmental effects
   + It's unclear how genetic structure can influence
 - Critical Analysis of Major Findings
   + This is the largest genetic analysis of *S. sclerotiorum* to date
   + Similar to the findings of @otto-hanson2011variation, we found that
     Region has a strong influence on both population structure and
     aggressiveness.
   + North Dakota appeared
 - Additional Findings and how they fit into the literature
 - Limitations
 - Future Directions
 - Overall Conclusion and Impact
-->



In this study, we characterized the diversity of *Sclerotinia sclerotiorum* from dry bean fields across the United States.
Our results suggest that, broadly, populations from white mold screening nurseries reflect the populations of the surrounding regions, indicating that resistance screening may be successful at a within regions. 
We found significant population differentiation by geographic region, mainly differentiated into three broad North American groups based on shared genotypes and posterior groupings, a Coastal Region, Midwestern Region, and Mexico. 
To date, with 366 isolates, this is the largest single population genetic study of *S. sclerotiorum* assessing population structure within managed and unmanaged agricultural environments. 
These findings indicate that the wmns can be effective at screening for potential resistant lines within growing regions. 

## Variable Importance {-}

Borrowing a technique often used in the ecological literature, we used dbRDA to elucidate the effect of all variables (Aggressiveness, Region, Source,   Year, and Host) [@legendre1999distance].
From the initial results, it appears that the most important factors for predicting genetic structure are MCG, region, and year.
When we inspected the biplot of the initial results, we saw that the most important predictors were MCGs 44, 5, and 9. 
We believe that this is driven by the fact that these particular MCGs have uneven MLG distributions, meaning that they are heavily associated with one particular MLG (Fig. \@ref(biggraph)).
We note these results with caution because of the apparent multicolinearity between MCG and Region, which is a violation of the analysis [@legendre1999distance].
While the results indicated that Mexico and the year 2005 were the two most important variables, it's worth noting that all Mexican isolates were collected in 2005 (Fig. \@ref(rda-biplot)).
The results also show that the Vista cultivar explains some of the variance, but this represents six isolates in MI. 
Aggressiveness and specific source field had little to no effect on genetic diversity. 
These results are in agreement with studies that examined differentiation based on host [@aldrich-wolfe2015genetic] and aggressiveness [@atallah2004high; @attanayake2012genetic; @attanayake2013sclerotinia] reporting little or no correlation of genetic diversity to these variables. 
This indicates that a) breeders should keep in mind regional differences when assessing resistance and b) it is possible that we have not yet measured biologically relevant variables that can predict genetic differentiation, which could include variables such as soil community composition. 

## Mycelial Compatibility Groups and Multilocus Genotypes {-}

Of interest in this study was testing the association between multilocus genotypes (MLGs) and mycelial compatibility groups (MCGs), as the latter have historically been used as a proxy for clonal lineages [@leslie1993fungal; @otto-hanson2011variation; @kohn1990mycelial; @schafer2006optimized; @kohn1995clonal; @carbone1999patterns]. 
Our results do not support this assumption.
It can be seen in Fig. \@ref(biggraph)A that the most abundant MCG contains several MLGs, but the diversity of those MLGs are low as indicated by the evenness (transparency), which indicates that there is one dominant MLG ('MLG 25').
What is not shown in Fig. \@ref(biggraph)A is the MLGs that are shared between MCGs. 
This is illustrated in both table \@ref(tab:mlg-table) and Fig. \@ref(biggraph)B,C. 
It could be argued, however that 'MLG 25', with its high value of $P_{sex}$ represents different true MLGs across the five MCGs it occupies, but this doesn't account for the overall structure of Fig. \@ref(fullgraph) where, for example, 'MLG 75' ($P_{sex}$ = 1.81e^-4^) is compatible with 58 other genotypes through three MCG when the population structure of *S. sclerotiorum* is known to be clonal.

Over the past few years, researchers have noticed inconsistencies among the relationship between MCGs and MLGs [@aldrich-wolfe2015genetic; @carbone1999patterns; @attanayake2012genetic; @lehner2015genetic]. 
Either several MCGs belong to one MLG, which could be explained by insufficient sampling of loci; several MLGs belong to one MCG, which could be explained by clonal expansion; or a mixture of both.
Some studies have shown a correlation between MCG and MLG [@aldrich-wolfe2015genetic; @carbone1999patterns; @lehner2015genetic], whereas other studies have shown no apparent correlation, even on small spatial scales [@atallah2004high; @attanayake2012genetic; @attanayake2013sclerotinia].

One long-held assumption was that MCGs (as determined via barrage reaction) represent vegetative compatibility groups (VCGs) [@schafer2006optimized; @kohn1990mycelial; @lehner2015genetic], which are known to have a genetic component [@saupe2000molecular; @hall2010evolution; @strom2016genomes].
While our results appear to contradict this assumption, our protocol for assessing MCGs utilized a different substrate for these reactions and, thus may not accurately reflect the true MCG dynamics. 
However, it has been demonstrated in several Ascomycetes---including *Neurospora crassa* [@micali2003independence], *Sclerotinia homoecarpa* [@jo2008reassessment], *Verticillium dahliae* [@papaioannou2014barrage], and *S. sclerotiorum* [@ford1995heterokaryon]---that barrage reactions are independent from stable anastomosis.
Thus, the inconsistencies in this study and other studies indicate that researchers studying *S. sclerotiorum* should not rely on MCG data derived from barrage reactions as an indicator for genetic diversity.  
 
## Genotypic Structure {-}

The results show several genotypes shared across populations, except for Mexico and two states that had fewer than three samples each (Fig. \@ref(community-graph)).
Our network-based approach by treating the genotypes as edges and weighting each edge with the inverse of $P_{sex}$ treated the edges as springs connecting the populations with the strength proportional to the probability of obtaining the same genotype as a clone. 
This allowed us to use a graph walking algorithm to see how close the populations were simply based off of the proportion of clones they shared.
The most abundant genotype was shared across four populations, but its high value of $P_{sex}$ meant that it did not contribute significantly to the overall structure. 
The graph walking algorithm was able to divide the network into three groups, but had a modularity of 0.17, which indicates that the groups are only weakly differentiated.
This indicates that multilocus genotypes are widespread across the United States, giving more evidence for a clonal population structure.

## Population Differentiation {-}

The primary interest of this study was to assess if isolates sampled from white mold screening nurseries represent isolates from producer fields within the region [@steadman2003identification; @otto-hanson2011variation].
According to our AMOVA results, we have evidence for differentiation at the Region and Year, but little to no differentiation between wmn isolates and production field isolates (Table \@ref(tab:amova)).
When we analyze the AMOVA results in light of the DAPC results (Fig. \@ref(DAPC)), it becomes clear that the regional patterns of differentiation are largely driven by isolates from Mexico and CA.
Isolates from these Regions had a higher posterior probability (> 0.75) of being reassigned to their own populations than any other (Fig. \@ref(DAPC)a). 
All other populations in comparison (except France) has reassignment probabilities of < 0.5, which is reflected in the failure of the first two discriminant functions to separate these populations (Fig. \@ref(DAPC)b).

Despite the evidence that Mexico and CA contributing to a large amount of the population differentiation, Region like WA still had a large amount of internal variation (Supplementary Information).
The two distinct clusters for the WA Region showed that the 2008 population appeared different and, under further investigation, we found that the genotypes from this population were shared between CA and France (Fig. \@ref(DAPC-RY), \@ref(DAPC-RY-FULL), \@ref(community-graph)). 
From the DAPC, we extracted 5 loci that contributed >10% of the observed variation to either of the first two discriminant axes and found that the alleles with an allele frequency of >50% in 2007 were reduced to < 10% in 2008, replaced by an allele that was > 2 mutational steps away (Supplementary Information).
This could be due to several reasons which include mis-labeling of source material or introduction of migrant sclerotia in plant material.
Except for the WA Region, populations that were sampled across several years appeared to be relatively stable over time with overlapping distributions in the DAPC (i.e. NE and MI, Fig. \@ref(DAPC-RY)). 
DAPC is based on the principal components of Euclidean distance between genotypes [@jombart2010discriminant]. 
Unlike Bruvo's distance, Euclidean distance does not take into account the magnitude of the difference between alleles, which could inflate the distance measure in the presence of private alleles [@bruvo2004simple]. 
We found no evidence of private alleles in the Mexican and Californian isolates, suggesting that this pattern separating these populations from the others was not an artifact. 

It has been proposed within the *S. sclerotiorum* literature that isolates from sub-tropical and tropical climates are differentiated or more variable than populations from temperate climates [@lehner2017sclerotinia;@carbone2001multilocus; @attanayake2013sclerotinia]. 
This has been attributed to the notion that the fungus has the chance to undergo more reproductive cycles in the warmer climate [@carbone2001multilocus; @attanayake2013sclerotinia]. 
Populations in sub-tropical regions of China have been found to be more variable, sexually reproducing, and unrelated to populations in temperate regions of the United States [@attanayake2013sclerotinia].
The differentiation between the Mexican isolates and all other regions gives some evidence for this hypothesis as the only isolates in our study from sub-tropical regions are in the Mexican populations. 

We were unable to find evidence for sexual reproduction in any region. 
While the Australian population had a non-significant value of $\bar{r}_d$---which would suggest that we cannot reject the null hypothesis of random mating---the sample size was insufficient from which to draw conclusions [@agapow2001indices; @milgroom1996recombination].


## Aggressiveness {-}

We observed significant differences in aggressiveness based on region (Fig. \@ref(aggressiveness-boxplot), Table \@ref(tab:region-aggressiveness)). 
These results show a similar pattern to what was found previously with the exception of North Dakota, which increased in mean aggressiveness from 5 to 5.77 [@otto-hanson2011variation].
This increase was due in part to new data from producer field isolates collected after the previous study.
Two caveats should come with interpreting these data, however.
First, these straw tests were performed by a different person for these later isolates, which could suggest a more lenient or strict scoring system. 
Moreover to this point, many of the ND isolates fell within the 6--7 range, which denotes a physical boundary (disease symptoms around the second node) between intermediate and susceptible [@otto-hanson2011variation]. 
Second, these results are all within the intermediate range (4--6) of aggressiveness, which may not represent a significant biological difference.


## Limitations {-}

One of the main limitations of this study is the focus on *P. vulgaris* as a host. 
It has been shown that *S. sclerotiorum* in the midwestern United States does not have a particular preference for host [@aldrich-wolfe2015genetic]. 
If the distribution of *S. sclerotiorum* is even across agricultural hosts in the USA, then our sample may yet be representative of the genetic pool present in other crops and weedy species. 

Another limitation was the microsatellite markers used for this particular study [@sirjusingh2001characterisation].
The genotype accumulation curve showed no indication of a plateau, indicating that if we had sampled more loci, we would have resolved more multilocus genotypes. 
While 16 loci showed us similar results and began to show a plateau for the genotype accumulation curve, we were unable to use these results due to our uncertainty in the allele calls for these five extra loci. 
With the availability of an optically-mapped genome [@derbyshire2017complete], future studies describing the genetic diversity of *S. sclerotiorum* should employ techniques such as Genotyping-By-Sequencing [@davey2011genome], Sequence Capture [@grover2012targeted], or Whole Genome Sequencing.

## Conclusions {-}

eh.

<!-- 
Our study was conducted to characterize *S. sclerotiorum* isolates from different geographic areas to determine important phenotypic variables influencing genotypes; to ascertain reproductive behavior of the pathogen; to test whether different sampling years or locations have any effect on genotype/sexuality; and to identify abundant and unique genotypes across isolates and geographies.
The dbRDA revealed region, year, and host as the most influential phenotypic variables on genotypes.
However, further molecular analysis of pathogen isolates from selected hosts did not show any significant genetic difference among populations.
According to dbRDA, aggressiveness and location of isolates had little or no effect.
Previous studies on *S. sclerotiorum* from alfalfa soil [@attanayake2012genetic] and potato [@atallah2004high], and canola [@attanayake2013sclerotinia] plants reported little or no correlation of genotypes on pa25thogen aggressiveness.
This is important information for resistant breeding since genotypes of *S. sclerotiorum* isolates seem to depend to a certain degree on different levels of region and also on environmental conditions of the sampling year.
For instance our analysis indicated that MX, 2009, and 2005 were highly influential predictors of *S. sclerotiorum* genotypes (Fig. 7).
Conversely, France and Australia, which are geographically distant to North America, were not within the top 10 predictors of genetic composition of isolates (Fig. 7).
According to dbRDA, Mexican region imposed the highest impact on microsatellite loci of *S. sclerotiorum* isolates.
We used Bruvo's distance for calculating genetic distances of different isolates since it incorporates stepwise mutation model and applicable to microsatellite analysis.

The 366 isolates of our study belonged to 87 MCGs and 165 MLGs.
MCGs with multiple isolates were represented by several to many microsatellite genotypes and demonstrated lack of similarity between MCGs and microsatellite genotypes.
For instance, MCG 5 with the highest number of isolates (N = 73) represented 37 MLGs when characterized by 11 microsatellites.
The most common genotype among these 73 isolates was MLG 25 with 19 isolates.
Similarly, second most common MCG with 44 isolates was represented by 19 genotypes.
Some studies on *S. sclerotiorum* found a strong association between MCG and multilocus genotypes [@aldrich-wolfe2015genetic] while others reported no evidence for such consistent association [@atallah2004high; @attanayake2012genetic] or either reported an association for a part of the sample [@kohn1995clonal].
@attanayake2013sclerotinia carried out a genetic diversity study of *S. sclerotiorum* populations from canola in China and USA.
They reported the MCGs were in accordance with the microsatellite MLGs in the U.S. population but not in the Chinese population.
A study conducted on Canadian field populations of cultivated canola revealed MCGs and RFLP fingerprinting of *S. sclerotiorum* isolates were associated but MCGs of Norwegian isolates from wild perennial plant Ranunculua ficaria did not show such an association [@kohn1995clonal].
@aldrich-wolfe2015genetic characterized 145 isolates from four different host crops in north central region of the USA.
Isolates belonging to same MCG either shared identical genotypes or differed at no more than 2 of the 12 microsatellite loci.
They reported only 14% of MCGs contained isolates with different genotypes.
Different results were reported by @attanayake2012genetic, who characterized 40 *S. sclerotiorum* isolates from 1 m^2^ area of alfalfa seed production soil using microsatellite markers.
There were 15 MCGs with seven representing just a single isolate.
Though single-isolate MCGs contained distinct multilocus genotype, they also found several instances where MCGs and microsatellite MLGs did not similarly correspond.
Some isolates of the same multilocus genotype belonged to different MCGs.
Our results agree with this type of relationship between MCGs and MLGs.
Similarly, @atallah2004high did not find any strong correlation between microsatellite haplotypes and MCGs in 167 *S. sclerotiorum* isolates from four potato fields in the Columbia basin of WA state.
These 167 isolates belonged to 82 MCGs with 64 single-isolate MCGs.
We came across 43 single-isolate MCGs among 366 isolates from dry bean cultivars. 

Our study did not find any multilocus genotype/s common across all or majority of sampling geographies. <!-~ ZNK: I think this statement is a bit misleading. Yes, we didn't find any individual MLGs, but 46% of these are present across at least two populations.  ~->
The most widespread and common genotype (MLG 25, Supplemental Table 1) was found only in five states of the USA with majority of isolates confined to ND.
However, we did find some MLGs of France and Australian isolates to be present in the USA samples.
Some studies on *S. sclerotiorum* did not detect any shared MLGs between geographically distant populations.
@attanayake2013sclerotinia were unable to detect any shared microsatellite MLGs in *S. sclerotiorum* populations from infected canola fields in China and USA.
Isolate collections were limited to a single county in China whereas USA isolates were from ND state only.
This may have limited the chance of discovering genotypes common to both regions.
@atallah2004high examined the population structure of *S. sclerotiorum* in different potato fields of Columbia basin and reported only 15 isolates shared the most common microsatellite haplotype in a sample of 167.
Similarly, several other studies did not detect common fingerprints of *S. sclerotiorum* isolates in geographically distant populations: Australia and North America [@ekins1999genetic];  California, Washington, and Ontario [@malvarez2007new]; Canada, North Carolina, and Louisiana [@cubeta1997clonality]. 

Contrary to above results, @aldrich-wolfe2015genetic who did a survey on *S. sclerotiorum* from different host crops in the North Central states of the United States found a single microsatellite haplotype to be prevalent throughout the region.
A study in the UK that examined this pathogen from 12 populations in different crop plants and weed species *Ranunculus acris* (meadow buttercup) also reported one microsatellite haplotype was widely distributed across hosts and areas [@clarkson2013population].
Reason for our isolates from dry bean not to show a widespread MLG may be due to extensive sexual outcrossing and movement sclerotia by human activities.
Though conditions of outcrossing for *S. sclerotiorum* is not well studied, it is believed weather and agricultural conditions play a role in production and release of ascospores of this pathogen [@huang1991induction; @sexton2006population].
One study found release of spores from apothecia declined over 36- 168 H at low RH of 65-75% but not at high RH of 90-95% [@clarkson2003ascospore].
The same study reported increased ascospores discharge at 20-25$^{\circ}$ C compared to 15$^{\circ}$ C and the survival of *S. sclerotiorum* ascospores was reduced at high temperature and RH.
Our samples were collected from 2003 to 2012 and except for the white mold screening nurseries, came from different fields.
Different agro-ecological conditions prevailed during sampling seasons may have resulted in different rates of sexual outcrossing of the pathogen and hence reduced the chance of extracting a common genotype.
When conditions are favorable for ascospore production, release, and survival, co-infection of a single site on a plant by multiple ascospores is possible [@sexton2006population].
This may eventually result in facilitating outcrossing and recombination.

Interestingly, all Mexican genotypes were unique to that region.
On the other hand CA genotypes were present in the West Coast states of OR and WA which may be due to movement of sclerotia with planting material. <!-~ ZNK: Do we have some sort of evidence for this, or is this pure speculation? ~->
<!-~ 
I'm not sure what the point of this paragraph is.
I believe I can put the discussion of the graph results here. 
~->
Surprisingly, France and Australian genotypes were also isolated in the USA.
MLG 4 (see supplemental Table 1) was recorded from France, Australia and MN while France and USA isolates had four common MLGs.
Similarly, five MLGs were recorded common to both Australia and USA isolates.
However, there were six unique genotypes in clone-corrected France population (N= 6/14) and three genotypes in CA population (N = 3/15).
MLGs from Australian region, our samples were limited to 6, which is not a sufficient number to make inferences.
Our results indicated that different continents were not a defining factor for microsatellite genotypes of *S. sclerotiorum* isolated from dry bean.
This may be due to involuntary movement of sclerotia by human activities and probably because of recent speciation of *S. sclerotiorum*. 
A phylogenetic study on Sclerotinaceae showed that most of the sclerotial lineages including different species of Sclerotinia had evolved relatively recently from a common ancestor [@carbone1993ribosomal].
Few multilocus genotypes were unique to each sampling state of the USA but most of them were common among several states (Supplemental Table 1).
For instance, Michigan state from which we had the largest collection of isolates (N = 62) consisted of 43 MLGs.
Of these, 32 MLGs were present in at least one of the 5 states of NE, ND, WA, CO, MN; 4 MLGs were present in France isolates; and 11 were unique to MI sites (see Supplemental Table 1 for unique MLGs in different regions).
When selecting *S. sclerotiorum* isolates for screening dry bean germplasm for disease resistance, plant breeders may find above information useful.
A study on genetic diversity of *S. sclerotiorum* from multiple host crops in the North central region of the USA revealed there was little evidence for genetic isolation of this pathogen by host crop or by distance [@aldrich-wolfe2015genetic].
Though our isolates didn't come from different crops, we had isolates from many resistant and susceptible dry bean cultivars, which also didn't show genetic differentiation according to AMOVA.
It seems Mexican pathogen isolates are different from the rest and warrants further investigation using a larger population.
Some France genotypes were common in USA isolates but at the same time 43% (6/14) of the France genotypes were unique to that region indicating white mold resistant cultivars produced in the USA might not work in that region. <!-~ ZNK: I'm not entirely sold on this conclusion. ~->
Geographic distance between different states of the USA seem not a barrier for gene transfer of *S. sclerotiorum* isolates though unique genotypes did prevail at a low frequency in each state. <!-~ ZNK: I don't know that this adds anything. ~->
The genetic diversity of regions with >10 isolates had similar values (0.43 - 0.59) except for Mexican isolates.
*h* of MX was 0.26 and indicated less exposure of sampling sites to outside inocula. <!-~ ZNK: Does this mean that there's heavy inbreeding here? ~->
Test for linkage disequilibrium showed evidence of clonal reproduction for MX isolates (Ia = 2.61, P.Ia = 0.001, $\bar{r}_d$ = 0.37, P.$\bar{r}_d$  = 0.001).
This may have been another reason for low gene diversity.

DAPC analysis by region followed by AMOVA corroborated the uniqueness of Mexican genotypes by grouping them away from the rest of the isolates (Fig. 8).
Other populations based on geography were unable to group in separate clusters except for the Californian population.
We tested whether genotypes of *S. sclerotiorum* populations were influenced by different dry bean host cultivars and year of sampling by DAPC and AMOVA.
There was no host specificity but year of sampling did show a significant impact on genotypes.
Other studies also reported host identity is no barrier for gene flow of this pathogen [@aldrich-wolfe2015genetic; @clarkson2013population].
@atallah2004high revealed statistically significant impact of the collection date of *S. sclerotiorum* isolates from four potato fields in the Columbia Basin on molecular variance.
But there was no impact of the year or location of collection.
We also did not detect location as a significant predictor of molecular variance but year had a significant impact.
Our samples were collected on a single day in each year from each field and therefore did not warrant for an analysis of date of sample collection. <!-~ BSA: Is this true, coauthors? ~->

We performed DAPC without assigning isolates to different populations based on sampling information to see the how isolates fall into clusters according to their genetic relatedness.
We decided the most appropriate number as six since the DAPC results had 87% (N = 13/15) of Mexican isolates in a single cluster (Fig. 9 cluster 3).
When number of clusters was assigned less than that MX isolates did not group separately while higher number of cluster assignment resulted in breaking down of other clusters in Fig. 9 into smaller groups.
DAPC analysis without population assignments revealed that in general geography is not a barrier for gene flow since isolates of different geographies from USA, Australia and France clustered together indicating genetic relatedness and possible sexual recombination within each region.
For instance France isolates were present in cluster 1, 2, 4 and 5.
As mentioned above only exception was the Mexican isolates.
@njambere2014genotypic studied *S. trifoliorum*, a close relative *S. sclerotiorum* and reported four cryptic genetic populations by Bayesian clustering analysis of microsatellite loci.
Those clusters had no correlation to geographic locations of samples.
Similarly, a cluster analysis of microsatellite data of *S. sclerotiorum* collected from 12 geographically distant host populations in  England and Wales resulted only two main clusters pooling isolates from different populations together [@clarkson2013population].
However, some other studies found *S. sclerotiorum* isolates from distant locations did differ significantly when molecularly characterized [@attanayake2013sclerotinia; @cubeta1997clonality; @ekins1999genetic @malvarez2007new].

There were isolates that shared the same MLG belonging to different MCGs.
For instance MLG 25 consisted of 27 isolates in 5 MCGs.
This phenomenon is suggestive of a recombination by sexual outcrossing [@njambere2014genotypic; @sexton2006population].  In general high level of MLG diversity; and lack of association of MCG and microsatellite genotype are evidence of *S. sclerotiorum* undergoing sexual outcrossing and movement of isolates due to agricultural practices [@njambere2014genotypic].
Our isolates had a total of 165 MLGs and they did not correspond well with 87 MCGs.
Linkage disequilibrium analysis of populations in white mold screening nurseries indicated the possibility of sexual outcrossing in some regions (ex.
CA, FR, OR and WA) during certain sampling years while most locations indicated clonal reproduction.
For example wmn of WA was sampled in 2003, 2004, 2005, and 2008 but only 2008 isolates had evidence of outcrossing.
Similar to our results many studies have reported primarily clonal reproduction of *S. sclerotiorum* populations with evidence of occasional outcrossing:  canola and sunflower pathogen populations in north central region of the USA [@aldrich-wolfe2015genetic]; cabbage population in North Carolina [@cubeta1997clonality]; Canadian canola and North Carolina cabbage populations [@kohli1998random]; and 12 populations from five crops in England and Wales [@clarkson2013population].
Contrary to above, one study from an experimental cabbage field plot in Louisiana, found a population of 16 isolates of *S. sclerotiorum* were strictly clonal [@cubeta1997clonality].
@atallah2004high studied the genetic structure of *S. sclerotiorum* in four potato fields in the Columbia basin and reported evidence of potential outcrossing of populations in two fields.
When all isolates were combined, there was significant linkage disequilibrium and hence the population was highly clonal.
Our study supports this phenomenon since linkage disequilibrium analysis revealed evidence of sexual outcrossing only in CA when populations were based on region.
However, when populations were sub divided according each year/region, we could detect evidence of sexual out crossing in some populations.
One reason for this is the overshadowing of low frequency genotypes of heterothallic recombination by a few successful genotypes of clonal nature [@atallah2004high].  

*Sclerotinia sclerotiorum* isolates from dry bean represented high microsatellite diversity and lack of association MCGs and microsatellite MLGs.
Analysis of molecular variance of populations based on region and sampling year showed low level of differentiation with majority of variance detected within populations.
These features are indicative of sexual outcrossing and perhaps human mediated migration of *S. sclerotiorum* in dry bean.
However, we also detected clones or identical microsatellite MLGs within and among populations indicating asexual reproduction.
Further, linkage disequilibrium and index of association calculations suggested *S. sclerotiorum* populations are largely clonal.
Although, there was low impact of region and year on population differentiation, we did not detect host cultivar as a barrier to gene flow of the pathogen.
Our study did not find a widespread genotype among sampled regions from dry beans but some Australian and France MLGs were found in the USA isolates.
The analysis of populations from same field over different years for linkage disequilibrium did not provide evidence for sexual outcrossing.
However, analysis of molecular variance did show populations were significantly different between some years (eg.
MI 2003 and 2005 populations; WA 2008 compared to both 2003 and 2005).
These results indicate some level of heterothallic recombination is taking place in these fields in addition to natural mutations.
An understanding of sexual behavior and ability of form new genotypes by *S. sclerotiorum* in dry bean cropping systems is useful for formulating disease management strategies. --> 



# Supplementary Information {-}

<!-- This part is necessary for adding the "S" before supplementary figures.
Will not work with HTML outputs, though. -->

\setcounter{table}{0}
\renewcommand{\thetable}{S\arabic{table}}
\setcounter{figure}{0}
\renewcommand{\thefigure}{S\arabic{figure}}

```{r table2, echo = FALSE, warning = FALSE, message = FALSE, results = "asis"}
suppressPackageStartupMessages(library("tidyverse"))
read_csv("../../results/tables/Region-aggressiveness.csv", 
         na = "NA", # spaces read in as spaces
         col_types = cols(.default = col_character())) %>%
  huxtable::as_huxtable(add_colnames = TRUE) %>%
  huxtable::set_number_format(row = huxtable::everywhere, col = 1, 0) %>% 
  huxtable::set_col_width(c(0.13, 0.25, 0.08)) %>%
  huxtable::set_align(row = huxtable::everywhere, col = 2, "right") %>%
  huxtable::print_md(max_width = 40)
```

Table: (\#tab:region-aggressiveness) Mean aggressiveness ratings for the Regions with greater than 5 samples; groupings according to 95% family-wise confidence interval.

\newpage


![
Graph showing complex associations between Mycelial Compatibility Groups (MCG) (dotted nodes) and Multilocus Genotypes (MLG) (full nodes) where the number in each node represents the MLG/MCG assignment. 
Node size reflect the number of samples represented by each node (circle). 
Edges (arrows) point from MLG to MCG where the weight (thickness) of the edge represents the number of samples shared. 
Node color represents the community assignment based on the walktrap algorithm with a maximum of four steps [@pons2006computing]. 
](../../results/figures/publication/full-graph.pdf){#fullgraph width=100%}


<!-- community graph figure -->

![
Network of populations (nodes/circles) and their shared multilocus genotypes (MLG) (edges/lines) genotyped over 16 loci. 
Each node is labeled with **name (number of MLGs shared/number of MLGs total).** 
The shade and area of the nodes are proportional to the number of unique MLGs within the node and the inner nodes are proportional to the number of private MLGs to the region (bottom legend). 
Each edge represents a single MLG where its thickness represents the number of populations that share the MLG and the shade represents the value of $P_{sex}$, or the probability of encountering that MLG from two independent meiotic events.
](../../results/figures/publication/mlg-16.pdf){#community-graph16 width=100%} 

<!-- end fig -->


<!-- DAPC figure 3: The figure strikes back-->

![
Scatter plot of Discriminant Analysis of Principal Components on Regions and Years showing temporal variation across all Regions. 
Points (text labels) represent observed individuals connected to the population centroids with ellipses representing a 66% confidence interval for a normal distribution. 
The center of each component is represented as black grid lines.
](../../results/figures/publication/dapc_region_year.pdf){#DAPC-RY-FULL width=100%}

<!-- end figure -->

\newpage
